packages/@okta/i18n/src/properties/login_ja.properties

```diff
@@ -644,12 +645,13 @@ password.complexity.minAgeHours = \u6700\u5F8C\u306B\u30D1\u30B9\u30EF\u30FC\u30
 # {0} is a number
 password.complexity.minAgeDays = \u6700\u5F8C\u306B\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u5909\u66F4\u3057\u3066\u304B\u3089\u5C11\u306A\u304F\u3068\u3082{0}\u65E5\u7D4C\u904E\u3057\u306A\u3051\u308C\u3070\u306A\u308A\u307E\u305B\u3093\u3002
 password.reset.verification = \u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u30EA\u30BB\u30C3\u30C8\u3059\u308B\u306B\u306F\u3001\u6B21\u306E\u3044\u305A\u308C\u304B\u306E\u8981\u7D20\u3067\u8A8D\u8A3C\u3057\u3066\u304F\u3060\u3055\u3044\u3002
+password.reset.revokeSessions = \u4ED6\u306E\u3059\u3079\u3066\u306E\u30C7\u30D0\u30A4\u30B9\u304B\u3089\u30B5\u30A4\u30F3\u30A2\u30A6\u30C8\u3057\u307E\u3059\u3002
 
 # The actual description consists of the password length message + one or more list elements
 # {0} is a number
 password.complexity.length = {0}\u6587\u5B57\u4EE5\u4E0A
 # {0} refers to a translated password requirement (denoted below), like "a lowercase letter" or "an uppercase letter"
-password.complexity.list.element = \u3001{0}
+password.complexity.list.element = , {0}
```

Comment:
Brought this up in a different PR, and Mau brought this up in Slack -- `,` should not be in the Japanese translation